pkg/controller/bundles/bundles_syncer.go

```diff
@@ -87,6 +95,26 @@ func (syncer *LeafHubBundlesSpecSync) Start(stopChannel <-chan struct{}) error {
 	return nil
 }
 
+func readK8sPoolSizeEnvVar(log logr.Logger) int {
+	envK8sClientsPoolSize, found := os.LookupEnv(envVarK8sClientsPoolSize)
```

Comment:
can be called k8sClientsPoolSize (no need for the env prefix)

Reply:
done